package.json

```
@@ -55,9 +59,14 @@
     "node-sass": "^4.14.1",
     "sass-loader": "^6.0.6",
     "style-loader": "^2.0.0",
-    "webpack": "^4.46.0"
+    "webpack": "^4.46.0",
+    "webpack-cli": "^4.9.1"
   },
   "scripts": {
+    "test:e2e": "cross-env WP_BASE_URL=http://localhost: 9191 wp-scripts test-e2e --config=tests/e2e/jest.config.js",
```

```suggestion
    "test:e2e": "cross-env WP_BASE_URL=http://localhost:9191 wp-scripts test-e2e --config=tests/e2e/jest.config.js",
```

URL is incorrect here, There should be no space between port number